python/cudf/cudf/utils/queryutils.py

```diff
@@ -194,6 +194,8 @@ def query_execute(df, expr, callenv):
         Contains keys 'local_dict', 'locals' and 'globals' which are all dict.
         They represent the arg, local and global dictionaries of the caller.
     """
+    from cudf import MultiIndex
```

Comment:
Don't import in the function

Reply:
cudf isn't imported in the file, so I might have to import cudf before that. So not sure if that's a better option.

Reply:
Importing cudf is fine. In general we've seen that if these are used in tight loops they get really expensive versus just the module loading once.